src/fidesops/util/collection_util.py

```diff
@@ -34,3 +34,10 @@ def partition(_iterable: Iterable[T], extractor: Callable[[T], U]) -> Dict[U, Li
     for t in _iterable:
         append(out, extractor(t), t)
     return out
+
+
+def filter_nonempty_values(d: Dict[Any, Any]) -> Dict[Any, Any]:
+    """Return the input map with empty values removed"""
+    if d:
```

Comment:
if we're checking for `d`, is the input param `d: Optional[Dict[Any, Any])`?

Reply:
Turns out it's never being called with None. That being said, this is a good suggestion.